Sources/BraintreeVenmo/BTVenmoDriver.m

```diff
@@ -151,10 +151,23 @@ - (void)tokenizeVenmoAccountWithVenmoRequest:(BTVenmoRequest *)venmoRequest comp
                 }
             };
 
-            [self.apiClient POST:@"" parameters:params httpType:BTAPIClientHTTPTypeGraphQLAPI completion:^(BTJSON *body, __unused NSHTTPURLResponse *response, __unused NSError *err) {
-                // TODO add handling for `err` and if parsing `venmoPaymentContext id` fails.
+            [self.apiClient POST:@"" parameters:params httpType:BTAPIClientHTTPTypeGraphQLAPI completion:^(BTJSON *body, __unused NSHTTPURLResponse *response, NSError *err) {
+                if (err) {
+                    NSError *error = [NSError errorWithDomain:BTVenmoDriverErrorDomain
+                                                         code:BTVenmoDriverErrorTypeInvalidRequestURL
+                                                     userInfo:@{NSLocalizedDescriptionKey: @"Failed to fetch a Venmo paymentContextID while constructing the requestURL. Please contact support."}];
```

Comment:
```suggestion
                                                     userInfo:@{NSLocalizedDescriptionKey: @"Failed to fetch a Venmo paymentContextID while constructing the requestURL."}];
```

I know we do this in a few other places, but I'm not sure it makes sense to tell merchants to contact support here. If the request fails, they should just try again.